packages/examples/src/config.ts

```diff
@@ -0,0 +1,65 @@
+import { registerExamples } from './register';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    postalCode: {
+      type: 'string',
+      maxLength: 5
+    }
+  }
+};
+
+export const uischema = {
+  type: 'VerticalLayout',
+  elements: [
+    {
+      type: 'HorizontalLayout',
+      elements: [
+        {
+          type: 'Control',
+          scope: '#/properties/postalCode',
+          label: '%postalcode'
+        }
+      ]
+    }
+  ]
+};
+
+export const data = {
+  postalCode: '12345'
+};
+
+const translations = {
+  'en-US': {
+    postalcode: 'Postal Code'
+  },
+  'de-DE': {
+    postalcode: 'Postleitzahl'
+  }
+};
+
+const config = {
+  restrict: false,
```

Comment:
we should use non default values to show off the config

Reply:
should be both `true`